packages/mdc-textfield/foundation.js

```diff
@@ -65,9 +65,15 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
   }
 
   init() {
+    const input = this.getNativeInput_();
+
     this.adapter_.addClass(MDCTextfieldFoundation.cssClasses.UPGRADED);
     this.adapter_.registerInputFocusHandler(this.inputFocusHandler_);
     this.adapter_.registerInputBlurHandler(this.inputBlurHandler_);
+    this.registerAutoCompleteHandler_(input)
+      .then((element) => {
```

Comment:
do you need to have `(element)` as an arg here? Or can you just do `(() => this.activateFocus_())` ?